### PR TITLE
Add trace_log into stateless/stress test artifacts

### PR DESCRIPTION
**Pull request by azat**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Review by alexey-milovidov — APPROVED** [review]
LGTM, but I don't understand why do we need both text dump and filesystem snapshot.
Text dump should be enough.

**Comment by azat** [comment]
>I don't understand why do we need both text dump and filesystem snapshot.

I did it in the same way as other tables.

> Text dump should be enough.

I agree that one format is enough, but how about keeping filesystem snapshot instead of tsv?
Since the snapshot you can just copy and it will work, while tsv you need to import.

Also note, right now snapshots takes more space them tsv, because tsv compressed with gzip, so looks like snapshots should be .tar.gz to decrease space usage.

**Comment by azat** [comment]
trace_log size:
- snapshot: 33M
- tsv: 491K

**Comment by alexey-milovidov** [comment]
> I did it in the same way as other tables.

I remind it was for only one very special case when something was specific to query_log table. @alesapin maybe you know about it.

> trace_log size:

Let's leave tsv only.

**Comment by azat** [comment]
>Let's leave tsv only.

I did it the same as it done for other tables:
- stateless: tsv only
- stress: snapshot only

I can convert everything to tsv only if that make sense.

**Comment by akuzm** [comment]
It's not very useful if you don't resolve and save the adresses at the same time. Something like this: https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/performance-comparison/compare.sh#L321

**Comment by alexey-milovidov** [comment]
+1, it's easy to add addresses in SELECT.

File: docker/test/stateless/run.sh [hunk 1]
```diff
@@ -104,6 +104,12 @@ clickhouse-client -q "system flush logs" ||:
 pigz < /var/log/clickhouse-server/clickhouse-server.log > /test_output/clickhouse-server.log.gz &
 clickhouse-client -q "select * from system.query_log format TSVWithNamesAndTypes" | pigz > /test_output/query-log.tsv.gz &
 clickhouse-client -q "select * from system.query_thread_log format TSVWithNamesAndTypes" | pigz > /test_output/query-thread-log.tsv.gz &
+clickhouse-client --allow_introspection_functions=1 -q "
+    WITH
+        arrayMap(x -> demangle(addressToSymbol(x)), trace) AS trace_array,
```
Comment (alexey-milovidov) [inline docker/test/stateless/run.sh:109]: You may also want to add `addressToLine`.

**Pull request merged**